benchmark.py

```diff
@@ -201,3 +206,17 @@ def load_batch_to_device(batch, device):
 		float(args.B * example_time * args.iterations / times_fwd.sum())
 	)
 )
+
+if args.output:
+	with open(args.output, 'a+') as f:
+		template = f'{args.B}\t' \
```

Comment:
This is not a template, since it has no substitution fields. Plase rename it to `message` or `msg`

Reply:
done